Sources/SwiftParser/Statements.swift

```diff
@@ -1009,22 +1013,24 @@ extension Parser.Lookahead {
         // yield statement of some singular expression.
         return !self.peek().isAtStartOfLine
       }
-    case .forgetKeyword?:
+    case .forgetKeyword?:  // NOTE: support for deprecated _forget
+      fallthrough
+    case .discardKeyword?:
```

Comment:
Same here. `fallthrough` is discouraged in Swift

```suggestion
    case .discardKeyword?, .forgetKeyword?:  // NOTE: support for deprecated _forget
```